### DISCUSSION TITLE
Permissions to view pods grouped by node

### DISCUSSION THREAD

**Discussion by QuingKhaos** [argoproj/argo-cd#11969] [Q&A]
What permissions are necessary to allow users to view the pods grouped by node in the app UI?

ArgoCD k8s cluster is in namespaced mode.

**Comment by jannfis** [comment]
I believe the view would require Kubernetes RBAC permission on the `Node` resources, which are cluster-scoped in K8s.

In namespace mode, Argo CD usually has no permission to access cluster-scoped resources.
